panacea/query_client.go

```diff
@@ -40,8 +44,9 @@ type QueryClient struct {
 
 // NewQueryClient set QueryClient with rpcClient & and returns, if successful,
 // a QueryClient that can be used to add query function.
-func NewQueryClient(ctx context.Context, chainID, rpcAddr string, trustedBlockHeight int, trustedBlockHash []byte) (*QueryClient, error) {
-	rpcClient, err := rpchttp.New(rpcAddr, "/websocket")
+func NewQueryClient(ctx context.Context, config *config.Config, trustedBlockHeight int, trustedBlockHash []byte) (*QueryClient, error) {
```

Comment:
You can use `TrustedBlockInfo` struct in line 34 of this file :)

BTW, when creating new query client, can the trusted block info be retrieved from sgx level DB that you developed? not as args.

Reply:
Oh, thanks. I will change to use `TrustedBlockInfo`.

There is a part that checks whether the light block is stored before in the levelDB when `light.NewClient` is called. So, it can be retrieved. I test it in the sgx environment as well. Does this answer the question right?

Reply:
I fixed it in [8d45752](https://github.com/medibloc/panacea-doracle/pull/12/commits/8d4575274a8eaf1656b1f5e99f7364dcaca8d27f).

Reply:
As I understand, the trusted block info is given by user only when registering oracle. 
At this time, the trusted block info should be sealed and stored to be utilized for light client. (I also missed to implement this part)

That is, the trusted block info should only be managed by the light client with sgx level DB after registration, not user.
If the trusted block info is required when process starts, the oracle should get the trusted block info from the sgx level DB. Because the trusted block info must not be changed arbitrarily.

Reply:
@H4NLee As we said offline, I implement functions to store and retrieve trusted block info in levelDB. It is implemented in `panacea/sgxdb_util.go`. Thank you for checking again.